server/controllers/postController.ts

```diff
@@ -12,7 +12,12 @@ async function postPosts(req, res, next) {
 
 async function getPosts(req, res, next) {
 	if ('number' in req.query && 'page' in req.query) {
-		const result = await postManager.getPostsPage(req.query.number, req.query.page);
+		let result;
+		if ('sort' in req.query) {
+			result = await postManager.getPostsPage(req.query.number, req.query.page, req.query.sort);
+		} else {
+			result = await postManager.getPostsPage(req.query.number, req.query.page, 'default');
+		}
```

Comment:
You don't need this check. Just set the default value.
```suggestion
		const result = await postManager.getPostsPage(req.query.number, req.query.page, req.query.sort);
```